### PR TITLE
Navigation and avatar changes

### PR DESCRIPTION
**Pull request by arivepr**
This PR is to move in the changes made for cards 3085 and 8493. 
The PF Avatar is now all integrated with the dropdown menu and "Documentation" link has been moved to the bottom of the nav.

### CODE REVIEW

File: src/js/App/Sidenav/Navigation.js [hunk 1]
```diff
@@ -18,14 +18,14 @@ const openshiftLinks = {
         link: 'https://access.redhat.com/support/cases'
     },
     feedback: {
-        title: 'Cluster Manager Feedback',
+         title: 'Cluster Manager Feedback',
```
Comment (ryelo) [inline src/js/App/Sidenav/Navigation.js:21]: Extra space

File: src/js/App/Sidenav/Navigation.js [hunk 2]
```diff
@@ -18,14 +18,14 @@ const openshiftLinks = {
...
         link: 'mailto:ocm-feedback@redhat.com'
     },
     marketplace: {
         title: 'Red Hat Marketplace',
         link: 'https://marketplace.redhat.com'
     }
-};
+};
```
Comment (ryelo) [inline src/js/App/Sidenav/Navigation.js:28]: Extra space

File: src/js/App/Header/UserToggle.js [hunk 3]
```diff
@@ -79,10 +81,12 @@ export class UserToggle extends Component {
     render() {
         const { isOpen } = this.state;
         const { account, isSmall, extraItems } = this.props;
-        const toggle = isSmall ?
+        const toggle = isSmall 
+        ?
```
Comment (ryelo) [inline src/js/App/Header/UserToggle.js:85]: Move this back up to line 84

File: src/sass/chrome.scss [hunk 4]
```diff
@@ -95,6 +95,11 @@ aside {
         font-size: var(--pf-global--FontSize--xs);
     }
 
+    .ins-c-dropdown-icon {
+        height: 36px;
+        width: 36px;
+    }
+
```
Comment (ryelo) [inline src/sass/chrome.scss:102]: Can you move this to a new file `UserIcon.scss` and import it in `UserIcon.js`?

File: src/sass/chrome.scss [hunk 5]
```diff
@@ -106,6 +111,7 @@ aside {
     .pf-c-page__header-tools .pf-c-avatar {
         height: var(--pf-global--FontSize--4xl);
     }
+
```
Comment (ryelo) [inline src/sass/chrome.scss:114]: extra space

File: src/js/App/Header/UserIcon.js [hunk 6]
```diff
@@ -26,7 +26,7 @@ export class UserIcon extends Component {
     }
 
     render() {
-        const { avatar } = this.state;
+         const { avatar } = this.state;
```
Comment (ryelo) [inline src/js/App/Header/UserIcon.js:29]: Extra space

**Review by ryelo — DISMISSED** [review]

**Comment by ryelo** [comment]
@arivepr You can run `npm run lint` to generate these errors, but copying the lint:
```
/home/travis/build/RedHatInsights/insights-chrome/src/js/App/Header/Tools.js

  14:8  error  'UserIcon' is defined but never used  no-unused-vars

/home/travis/build/RedHatInsights/insights-chrome/src/js/App/Header/UserToggle.js

  8:10  error  'Avatar' is defined but never used  no-unused-vars
```

File: src/js/App/Header/Tools.js [hunk 7]
```diff
@@ -1,17 +1,13 @@
 import React, { useState, useEffect } from 'react';
-
-import { Button } from '@patternfly/react-core/dist/js/components/Button/Button';
+import { Button } from '@patternfly/react-core/dist`/js/components/Button/Button';
```
Comment (karelhala) [inline src/js/App/Header/Tools.js:2]: Extra character
```suggestion
import { Button } from '@patternfly/react-core/dist/js/components/Button/Button';
```

**Comment by karelhala** [comment]
Remove the extra ` and update snapshots and we are golden!

**Review by karelhala — APPROVED** [review]

**Comment by ryelo** [comment]
@karelhala sorry I added the `on hold` label because we were tweaking some loading states. We'll throw it in another branch 😄

**Comment by karelhala** [comment]
@ryelo too fast on the merge button 😁

**Pull request merged**